### DISCUSSION TITLE
I only want to see what repo's the people I follow star and create and who stars and follows me. Nothing else.

### DISCUSSION THREAD

**Discussion by sweetbbak** [community/community#66268] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

The new feed is awful. As I said in the title, I just want to see the people who I follows basic activity and my basic activity. Now I can't even do that. What the heck was Microsoft thinking here?

**Comment by sketchbuch** [comment]
+1

**Comment by twobiers** [comment]
I mean... you could use the filter.

**Reply by sketchbuch** [reply]
I mean... they could have let me switch manually to the "For you" feed if I want to see repos I starred 10 years ago. Also as other have said uising the filters results in an empty feed.

**Reply by port19x** [reply]
The filter gets you nowhere close to that. You will still have people in there you don't follow and don't care about

**Reply by pouriap** [reply]
You can't. I just used to filter to only show stars and follows and it doesn't show anything. It's just blank.

**Comment by sooftware** [comment]
+1

**Comment by wangrongding** [comment]
# Github Old Feed  

💪 Using this script you can restore the old version of the feed.
https://github.com/wangrongding/github-old-feed  

<table>
  <tr>
      <td width="50%" align="center"><b>Before</b></td>
      <td width="50%" align="center"><b>After use</b></td>
  </tr>
  <tr>
     <td><img src="https://assets.fedtop.com/picbed/1694066442147.jpg"/></td>
     <td><img src="https://assets.fedtop.com/picbed/1694066437621.jpg"/></td>
  </tr>
</table>

**Comment by foo123** [comment]
+1 where are the stars on our own repositories?

**Comment by klren0312** [comment]
+1

**Comment by bunnyi116** [comment]
+1
